packages/wallets/talisman/package.json

```diff
@@ -1,16 +1,19 @@
 {
   "description": "SwapKit Wallet - Talisman",
   "dependencies": {
-    "@polkadot/api": "10.12.6",
-    "@polkadot/rpc-provider": "10.12.6",
-    "@polkadot/types": "10.12.6",
     "@polkadot/util": "12.6.2",
     "@polkadot/util-crypto": "12.6.2",
     "@swapkit/helpers": "workspace:*",
     "@swapkit/toolbox-evm": "workspace:*",
     "@swapkit/toolbox-substrate": "workspace:*"
   },
-  "files": ["src/", "dist/"],
+  "devDependencies": {
+    "@polkadot/types": "12.1.1"
+  },
```

Comment:
This will be included via `toolbox-substrate` no? 🤔

Reply:
done